ts/components/FiscalCodeComponent.tsx

```diff
@@ -334,6 +334,9 @@ export default class FiscalCodeComponent extends React.Component<Props> {
             : [styles.fullText, fullStyle]
         ]}
         selectable={selectable}
+        accessible={true}
```

Comment:
I guess we need to add
`accessibilityRole={"button"}`
in `ts/screens/profile/ProfileMainScreen.tsx` line 534 to read "button" so the user can know he can access it by double tap

Reply:
@Undermaken should we consider it a "button" even if it's not properly a button? 
I have chosen to explain it by the hint `profile.fiscalcode.accessibility.preview.hint`: _Activate to show the facsimile of your fiscal code document_

what do you think?

Reply:
@CrisTofani I noted on iOS the hint is not pronounced when the accessibility label is defined. This is why I can't understand if that element could be activated 
Moreover the CF preview acts like a button (it is wrapped inside a TouchableDefaultOpacity )
https://github.com/pagopa/io-app/blob/836a9b1dd7159d1bb73e4ca9f17efc5258ceabc4/ts/screens/profile/ProfileMainScreen.tsx#L534

Reply:
fc19996 adds the role **button** on `preview` and `full` visualization and **none** on `landscape`